eZ/Publish/Core/MVC/Symfony/Cache/Http/LocationAwareStore.php

```diff
@@ -1,255 +0,0 @@
-<?php
```

Comment:
Hmm, we can add deprecation warnings, can't we ?

They will for sure notice if they were using it and we remove it :)